### ISSUE TITLE
Remove "Listen to page" screen reader from tab order

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble-AssemblyLine#621]
I thought this was already in, but I guess it's not.

We should remove the entire widget from the tab order, and probably from accessibility tree in general, so it's not announced. Right now, it's still announced and is in the tab order. IMO we should do both.

**Comment by nonprofittechy** [comment]
Is there any reason to allow people who are sighted but want to have the screen read aloud be able to tab to this button? Just wanted to flag that possibility.

**Comment by BryceStevenWilley** [comment]
It's a good point, and TBH I'm not super sure. We need to weight if it's more of a harm to low-vision users who are already using a screen reader, or if it's more of a benefit to keyboard control users who aren't also using a screen reader.

T&P advised taking it out entirely, and my understanding is that we decided not to since it's not functioning as an accessibility feature; it's solely for low literacy users who might not be familiar with screen reader technology. To my knowledge, there isn't a way to satisfy both groups: both not announcing it to screen readers and still being able to tab to the button (as then it would be more confusing, not announcing something that a low-vision user could interact with).

IMO, leaning on T&P's suggestion probably a better idea. Two voices reading the screen at the same time is really confusing for screen readers, and should be avoided.

**Comment by nonprofittechy** [comment]
Fair enough, as long as we have a thought out plan. Tab navigation by sighted folks is a relatively small use case and will never happen on mobile
